ios/TransparentVideoViewManager.swift

```diff
@@ -17,20 +17,27 @@ class TransparentVideoView : UIView {
 
   private var source: VideoSource?
   private var playerView: AVPlayerView?
+  private var videoAutoplay: Bool?
 
   @objc var src: NSDictionary = NSDictionary() {
     didSet {
       self.source = VideoSource(src)
+    }
+  }
+
+  @objc var autoplay: Bool = false {
+    didSet {
+      self.videoAutoplay = autoplay
       let itemUrl = URL(string: self.source!.uri!)!
       loadVideoPlayer(itemUrl: itemUrl)
     }
   }
```

Comment:
I'm not sure what order react native will set these variables in. I wonder if it's based on the order that the view props are set in the `TransparentVideoViewManager.m` file 🤔 . 

I worry if it sets `autoplay` first, with the force unwraps on line 31 we'd crash. I wonder if we should call loadVideoPlayer in both of these `didSet` scopes and early return in `loadVideoPlayer` if the url on the player is equal to the url we're trying to set. Basically make it do nothing if it gets called with the same url many times.

Reply:
oh that's pretty good, lemme try that.